lib/views/multiline.coffee

```diff
@@ -0,0 +1,21 @@
+class MessageLine extends HTMLElement
```

Comment:
There is a reason we dropped the `linter-panel` and `linter-inline` Custom Elements and replaced them with traditional divs. Redundant code should be avoided wherever we can.